var/spack/repos/builtin/packages/hicops/package.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hicops(Package):
+    """HiCOPS is a software framework for accelerating database peptide search
+    workflows on supercomputers. HiCOPS provided algorithm-independent
+    parallelizations and optimizations can be extended into new HPC database search
+    algorithms or scalably accelerate the existing ones.
+    """
+
+    homepage = "https://hicops.github.io/index"
+    git = "https://github.com/hicops/hicops.git"
+    maintainers = ['pcdslab', 'mhaseeb123', 'nessiecancode']
+
+    version('release', branch='release')
+    version('develop', branch='develop')
+
+    # Build Options
+    variant('USE_MPI', default='ON',
+            description='CMake Option: Enable MPI support.',
+            values=('ON', 'OFF'))
+    variant('USE_TIMEMORY', default='OFF',
```

Comment:
We don't map 1:1 the names that the build-system expects. In this case you can use e.g. `timemory` and a variant name.

Reply:
This variant should also add the `timemory` package as a dependency.

Reply:
I have adjusted the code to reflect this change.